example/service.py

```diff
@@ -1,22 +1,22 @@
 #!/usr/bin/env python3
 
 import time
-import requests
+import urllib.request
```

Comment:
Using the standard library for a smaller footprint, good. The built-in client is ok for this minimal usage. `from urllib.request import urlopen` would be more idiomatic.